code/modules/hydroponics/grown/apple.dm

```diff
@@ -31,7 +31,7 @@
 	bitesize = 0.01
 	var/list/eaters = list()
 	
-/obj/item/reagent_containers/food/snacks/grown/apple/poisoned/on_bite(mob/living/carbon/biter)
+/obj/item/reagent_containers/food/snacks/grown/apple/poisoned/on_bite(mob/living/carbon/human/biter)
```

Comment:
_monkey tries to eat poison apple_

_runtime_

Reply:
IT'S YOU! THE FIRST ACCOUNT OF DARKBITT! THIS IS TRULY AN HONOUR!﻿

Reply:
Let me give you some advice, stranger. Leave. Now. You ain't welcome here.

Reply:
I saw your fight against the Admins! You're the best! Can I... Can I follow you around? I won't get in the way!

Reply:
FEAR MY PEE DISPENSER, LOYAL-DOER


***if you don't get the reference, please don't go digging in the logs for it. plz no***

Reply:
> imagine being so young you don't get this PR's reference.

Reply:
Golly, you're the best! I'm going to follow you and watch you and worship the ground you walk on! Let's go!

Reply:
i don't waste no time watching no doctor seuss

Reply:
WHY. WON'T. YOU. DIE?!

Reply:
You elves are all the same.